analysis_options.yaml

```diff
@@ -50,7 +50,7 @@ analyzer:
     unnecessary_null_comparison: false
     always_require_non_null_named_parameters: false # not needed with nnbd
     # TODO(https://github.com/flutter/flutter/issues/74381)
```

Comment:
What is the todo here? Even looking at the issue, I have no idea what needs to be done in this file that the TODO is about.

Reply:
Oh I see, I missed that the setting is ignoring it, not enabling it. Still, it would be helpful if you could update this comment to explicitly say that the todo is to fix the violations and then enable this setting, rather than assume the reader has full context here.

Reply:
Apologies for the brevity; the TODO would be to clean up existing violations, and then remove the line to ignore `unnecessary_import`. Should I expand on that in the comment?